ReactiveCocoa/CocoaTarget.swift

```diff
@@ -4,16 +4,51 @@ import enum Result.NoError
 
 /// A target that accepts action messages.
 internal final class CocoaTarget<Value>: NSObject {
+	private enum State {
+		case idle
+		case sending(queue: [Value])
+	}
+
 	private let observer: Observer<Value, NoError>
 	private let transform: (Any?) -> Value
```

Comment:
`Any?` is the sender. `Value` is not necessarily the sender, but a property of it.